### PR TITLE
Animation Editor: Moved motion properties to unified inspector window

### PR DESCRIPTION
**Pull request by amzn-jillich**
Old:
image

New:
image

Signed-off-by: Benjamin Jillich <jillich@amazon.com>

### CODE REVIEW

File: Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionWindow/MotionWindowPlugin.cpp [hunk 1]
```diff
@@ -120,21 +121,22 @@ namespace EMStudio
...
         m_addMotionsAction                   = nullptr;
         m_saveAction                         = nullptr;
-        m_motionNameLabel                    = nullptr;
     }
 
 
     MotionWindowPlugin::~MotionWindowPlugin()
     {
-        delete m_dialogStack;
+        if (m_motionPropertiesWindow)
```
Comment (moraaar) [inline Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionWindow/MotionWindowPlugin.cpp:134]: no need to have a condition before deleting, delete call handles using `nullptr`.

**Review by moraaar — APPROVED** [review]
LGTM. Approved with one minor comment.

**Review by amzn-rhhong — APPROVED** [review]

**Review by hultonha — APPROVED** [review]
LGTM 👍 Only a couple of super minor gripes, nothing critical

File: Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionWindow/MotionExtractionWindow.cpp [hunk 2]
```diff
@@ -418,68 +403,49 @@ namespace EMStudio
...
-        //if (motionExtractionWindow->visibleRegion().isEmpty() == false) // TODO: enable this again. problem is we need some callback when it gets visible again and as this is not related to the window plugin but a stack dialog we first need to add some callback for "OnHeaderButtonPressed" inside the dialog stack...also let's not forget to add it to the plugin's VisibilityChanged callback then!
-        motionExtractionWindow->UpdateInterface();
-
-        return true;
     }
 
-    bool MotionExtractionWindow::CommandSelectCallback::Execute(MCore::Command* command, const MCore::CommandLine& commandLine)
+    bool MotionExtractionWindow::SelectActorCallback::Execute(MCore::Command* command, const MCore::CommandLine& commandLine)
     {
         MCORE_UNUSED(command);
```
[the remark this answers is not in the document]
Comment (amzn-jillich) [reply Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/Plugins/StandardPlugins/Source/MotionWindow/MotionExtractionWindow.cpp:414]: Changed

**Pull request merged**